src/main/java/org/codeforamerica/messaging/models/Template.java

```diff
@@ -0,0 +1,66 @@
+package org.codeforamerica.messaging.models;
+
+import com.github.jknack.handlebars.Handlebars;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.util.Map;
+import java.util.function.Function;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Template {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+    @NotBlank
+    String name;
+    String subject;
+    @NotBlank
+    String body;
```

Comment:
> Just a thought. We know we need language and experiment variants. Should we incorporate that in our data model design in this story or retread the path in the prototype (which may involve messy migrations)?

In the interest of avoiding migrations, I am considering adding separate fields for `sms_body` and `email_body` since we know we will want to support that in the future. The alternative is to have a `channel` field that signifies how the template is used, but to me that seems harder to track. We can discuss at standup.

Reply:
I am in favor of holding off as the migration in this case would be straightforward.

Reply:
I don't think the language and variants comment has been addressed.

Reply:
I have it locally but was holding off in case there were other additional fields too, but I guess that's not something to worry too much about

Reply:
I am adding `language` with a default of `en_US` and `variant` with default of `A`, thoughts on that? [This is the most useful list](https://help.sap.com/docs/SAP_BUSINESSOBJECTS_BUSINESS_INTELLIGENCE_PLATFORM/09382741061c40a989fae01e61d54202/46758c5e6e041014910aba7db0e91070.html) I could find of languages/locales. I am curious about how picky we should be about languages/locales/internationalization off the bat. E.g. for our spanish templates would we use the dominant locale `es_ES`, or the more relevant one, `es_MX`? I'm not completely sure of the implications of this, only that the subject can get very complicated.

Reply:
IMO, we could stay at the language level as we did for the prototype. I don't see a need for the "region" level at this point. It could be added easily if needed.